app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -371,29 +374,35 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
 /// Returns true if at least one approved review and no outstanding change
 /// request reviews.
 bool _checkApproval(
+  String? author,
+  String authorAssociation,
   List<Map<String, dynamic>> reviewNodes,
   Set<String?> changeRequestAuthors,
 ) {
   assert(changeRequestAuthors.isEmpty);
-  bool hasAtLeastOneApprove = false;
+  const Set<String> allowedReviewers = <String>{'MEMBER', 'OWNER'};
+  final Set<String?> approvers = <String?>{};
+  if (allowedReviewers.contains(authorAssociation)) {
+    approvers.add(author);
+  }
   for (Map<String, dynamic> review in reviewNodes) {
     // Ignore reviews from non-members/owners.
-    if (review['authorAssociation'] != 'MEMBER' && review['authorAssociation'] != 'OWNER') {
+    if (!allowedReviewers.contains(review['authorAssociation'])) {
       continue;
     }
 
     // Reviews come back in order of creation.
     final String? state = review['state'] as String?;
     final String? authorLogin = review['author']['login'] as String?;
     if (state == 'APPROVED') {
-      hasAtLeastOneApprove = true;
+      approvers.add(authorLogin);
       changeRequestAuthors.remove(authorLogin);
     } else if (state == 'CHANGES_REQUESTED') {
       changeRequestAuthors.add(authorLogin);
     }
   }
-
-  return hasAtLeastOneApprove && changeRequestAuthors.isEmpty;
+  final bool approved = (approvers.length > 1) && approvers.isNotEmpty;
```

Comment:
`(approvers.length > 1) && approvers.isNotEmpty` is redundant, you just need `approvers.length > 1`.

Reply:
Updated!